### PR TITLE
feat(generate-support-irversion): impl

### PR DESCRIPTION
**Pull request by zdunecki**
`irVersion` flag passed for `generate` command allows the use of explicit IR version, e.g `v32`.

### CODE REVIEW

File: packages/cli/cli/src/commands/generate/generateAPIWorkspace.ts [hunk 1]
```diff
@@ -61,6 +63,7 @@ export async function generateWorkspace({
             workspace,
             generatorGroup: group,
             keepDocker,
+            irVersion,
```
Comment (dsinghvi) [inline packages/cli/cli/src/commands/generate/generateAPIWorkspace.ts:66]: @zdunecki can we pass the `irVersion` in both local or remote ?

**Comment by dsinghvi** [comment]
@zdunecki looks great! if you wire it up with the remote code generation as well, that would be great!

File: fern/docs/pages/cli/fern-generate.mdx [hunk 2]
```diff
@@ -44,6 +44,17 @@ $ fern generate --version
...
 
+### Intermediate Representation(IR) Version
+
+**Type**: `string`
+**Description:** Specify a version of Intermediate Representation(IR).
+
+**Example**
+
+```bash
+$ fern generate --irVersion v32
```
Comment (dsinghvi) [inline fern/docs/pages/cli/fern-generate.mdx:55]: @zdunecki looks great -- thoughts on calling this `--irVersionOverride` to signify that you are overriding typical CI behavior ? 

For example if you run the python generator with `--irVersion 33` even though it actually depends on 32, the CLI would now allow it

**Comment by zdunecki** [comment]
@dsinghvi 

I've introduced your suggestions.

**Comment by dsinghvi** [comment]
Solved by https://github.com/fern-api/fern/pull/3212

**Pull request closed without merging**